src/GeoPoint.php

```diff
@@ -32,30 +32,24 @@ class GeoPoint implements JsonSerializable
      * @param float $lat Latitude
      * @param float $lng Longitude
      */
-    public function __construct($lat, $lng)
+    public function __construct($lat = 0, $lng = 0)
```

Comment:
we should change our doc comments as 

```
     * @param float|int $lat Latitude
     * @param float|int $lng Longitude
```

or we should change our parameters.